.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+# 2.1 does not yet support local run
+# unless with workaround. For simplicity just use it.
+# https://github.com/CircleCI-Public/circleci-cli/issues/79
+
+jobs:
+  build:
```

Comment:
If we are doing everything in the same job, I'd rename it to `test`

Reply:
I'd like to do it in a follow up commit if you will be OK. In that commit I will try to 
1. Split `build` into `build` and multiple stages of `build`, `lint`, `test`, `test_with_sloc_nightly`(if applies) and into workflow.
2. And then try to split the tests (currently around 2000~ish) into test_sets to allow parallelism 

How do you like it?

Reply:
That sounds great! I'd keep this PR focused on point 1. We can look into test parallelism in a separate PR, how does that sound?

Reply:
Cool

Reply:
It currently does not support local run for workflows in circleci-local